DigitalLearningSolutions.Web/ViewModels/LearningPortal/Current/MarkActionPlanResourceAsCompleteFormData.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+{
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Web.Helpers;
+
+    public class SetLearningLogItemCompletionDateFormData : IValidatableObject

```

Comment:
Looks like the class wasn't renamed but the file was.

Reply:
oops, renamed